docs/topics/getting-started-launchpad-update-booster-manual.adoc

```diff
@@ -1,6 +1,6 @@
 = Updating Manually
 
-This approach allows you to deploy updates directly from a local version of the GitHub repository created by {launcher}. 
+This approach allows you to deploy updates directly from a local version of the GitHub repository created by {launcher}.
```

Comment:
using a project created locally on your laptop from a Github repository

Reply:
If you accept this change, please replace _laptop_ with _machine_.

Reply:
@cmoulliard not sure how the wording you suggest is different. -- we're are saying this is a local version of the GitHub repository.